lib/tasks/setup.rake

```diff
@@ -12,7 +12,8 @@ namespace :redcap2omop do
        Redcap2omop::Person,
        Redcap2omop::ProcedureOccurrence,
        Redcap2omop::Provider,
-       Redcap2omop::Observation
```

Comment:
May be replace with Redcap2omop::Setup.omop_tables?

Reply:
Done.